config/jobs/kubernetes-sigs/apiserver-network-proxy/OWNERS

```diff
@@ -5,8 +5,10 @@ reviewers:
   - mcrute
   - anfernee
   - andrewsykim
```

Comment:
Could we also add imran and remove anfernee + andrewsykim?

Reply:
And remove mcrute

Reply:
Probably should do the owners change in a separate PR from the test change.

Reply:
Done, thanks.